web/impact/impact/v1/views/user_list_view.py

```diff
@@ -100,6 +106,44 @@ def _invalid_keys(self, user_keys):
         for key in set(user_keys) - set(ALL_USER_RELATED_KEYS):
             self.errors.append(INVALID_KEY_ERROR.format(key))
 
+    def _user_results(self, limit, offset):
```

Comment:
I've made the names consistent for now at some level we could pull things into a superclass but I'm going to leave that on the table for another time